pkg/crc/constants/constants_nonwin.go

```diff
@@ -0,0 +1,10 @@
+// +build !windows
+
+package constants
+
+import "os"
+
+// GetHomeDir returns the home directory for the current user
+func GetHomeDir() string {
```

Comment:
How is this different from https://golang.org/pkg/os/#UserHomeDir ? Could we use this instead?

Reply:
Yup, for the non-win case we can use this..

Reply:
Done!